stl/inc/flat_map

```diff
@@ -960,210 +886,530 @@ private:
-                          flat_multimap<_Key, _Mapped, _Compare, _KeyContainer, _MappedContainer>> {
+    // [flat.map.access] Access
+    _NODISCARD mapped_type& operator[](const key_type& _Key_val) {
+        return this->try_emplace(_Key_val).first->second;
+    }
+    _NODISCARD mapped_type& operator[](key_type&& _Key_val) {
+        return this->try_emplace(_STD move(_Key_val)).first->second;
+    }
+
+    template <class _OtherKey>
+    _NODISCARD mapped_type& operator[](_OtherKey&& _Key_val)
+        requires _Is_transparent_v<key_compare>
+    {
+        return this->try_emplace(_STD forward<_OtherKey>(_Key_val)).first->second;
+    }
+
+    _NODISCARD mapped_type& at(const key_type& _Key_val) {
+        return _At(_Key_val);
+    }
+
+    _NODISCARD const mapped_type& at(const key_type& _Key_val) const {
+        return _At(_Key_val);
+    }
+
+    template <class _OtherKey>
+    _NODISCARD mapped_type& at(const _OtherKey& _Key_val)
+        requires _Is_transparent_v<key_compare>
+    {
+        return _At(_Key_val);
+    }
+
+    template <class _OtherKey>
+    _NODISCARD const mapped_type& at(const _OtherKey& _Key_val) const
+        requires _Is_transparent_v<key_compare>
+    {
+        return _At(_Key_val);
+    }
+
+    // [flat.map.modifiers] Modifiers
+    template <class... _ArgTypes>
+    pair<iterator, bool> emplace(_ArgTypes&&... _Args)
+        requires is_constructible_v<value_type, _ArgTypes...>
+    {
+        value_type _Val(_STD forward<_ArgTypes>(_Args)...);
+        return try_emplace(_STD move(_Val.first), _STD move(_Val.second));
+    }
+
+    pair<iterator, bool> insert(const value_type& _Pair_val) {
+        return try_emplace(_Pair_val.first, _Pair_val.second);
+    }
+
+    pair<iterator, bool> insert(value_type&& _Pair_val) {
+        return try_emplace(_STD move(_Pair_val.first), _STD move(_Pair_val.second));
+    }
+
+    template <class _PairValTy>
+    pair<iterator, bool> insert(_PairValTy&& _Pair_val)
+        requires is_constructible_v<value_type, _PairValTy>
+    {
+        return emplace(_STD forward<_PairValTy>(_Pair_val));
+    }
+
+    template <class... _MappedArgTypes>
+        requires is_constructible_v<mapped_type, _MappedArgTypes&&...>
+    pair<iterator, bool> try_emplace(const key_type& _Key_val, _MappedArgTypes&&... _Mapped_args) {
+        return _Try_emplace(_Key_val, _STD forward<_MappedArgTypes>(_Mapped_args)...);
+    }
+
+    template <class... _MappedArgTypes>
+        requires is_constructible_v<mapped_type, _MappedArgTypes&&...>
+    pair<iterator, bool> try_emplace(key_type&& _Key_val, _MappedArgTypes&&... _Mapped_args) {
+        return _Try_emplace(_STD move(_Key_val), _STD forward<_MappedArgTypes>(_Mapped_args)...);
+    }
+
+    template <class _OtherKey, class... _MappedArgTypes>
+        requires _Is_transparent_v<key_compare> && is_constructible_v<key_type, _OtherKey>
+              && is_constructible_v<mapped_type, _MappedArgTypes&&...>

```

Comment:
Similarly, `is_constructible_v<mapped_type, _MappedArgTypes&&...>` should omit the `&&`. Occurs below.